### ISSUE TITLE
Manage HTTP redirects from the proxied server

### ISSUE THREAD

**Issue by giovannipizzi** [aiidalab/jhproxy#1]
Currently, if a HTTP redirect is given by the proxied server, this is proxied as well. 
This means that the browser will be redirected to things like `http://localhost:some_port/xxx` instead of just redirecting to `xxx` under the correct proxied path.

I think this just involves using some regex substitution as I think also e.g. apache reverse_proxy does.

**Comment by ltalirz** [comment]
This problem exists also in nbserverproxy?

**Comment by giovannipizzi** [comment]
no, I think the same problem occurs there, as the code is basically the same.

I don't think setting "allow_redirects" in the internal coroutine that does the request is the right solution either, because often redirects are meant to change the URL of the user, while that setting would hide the redirect. 

a link, maybe useful:

https://docs.trafficserver.apache.org/en/4.2.x/admin/reverse-proxy-http-redirects.en.html
